crates/neon-runtime/Cargo.toml

```diff
@@ -5,16 +5,20 @@ authors = ["Dave Herman <david.herman@gmail.com>"]
 description = "Bindings to the Node.js native addon API, used by the Neon implementation."
 repository = "https://github.com/neon-bindings/neon"
 license = "MIT/Apache-2.0"
+edition = "2018"
```

Comment:
Updated to edition 2018 because I only know how to use `macro_rules!` in this edition